scripts/makebumpver

```diff
@@ -36,16 +36,116 @@ import sys
+
+        parser.add_argument("-n", "--name", dest="name", default=current_name, metavar="PACKAGE NAME", help="Package name.")
+        parser.add_argument("-v", "--version", dest="version", default=current_version, metavar="CURRENT PACKAGE VERSION",
+                            help="Current package version number.")
+        parser.add_argument("-r", "--release", dest="release", default=1, metavar="PACKAGE RELEASE NUMBER",
+                            help="Package release number.")
+        parser.add_argument("--newrelease", dest="new_release", default=None, help="Value for release in the .spec file.")
+        parser.add_argument("-b", "--bugreport", dest="bugreporting_email", default=current_bug_reporting_mail, metavar="EMAIL ADDRESS",
+                            help="Bug reporting email address.")
+        parser.add_argument("-i", "--ignore", dest="ignored_commits", default=[], action=ParseCommaSeparatedList,
+                            metavar="COMMA SEAPARATED COMMIT IDS", help="Comma separated list of git commits to ignore.")
+        parser.add_argument("-m", "--map", dest="fedora_rhel_bz_map", default=[], action=ParseCommaSeparatedList,
+                            metavar="COMMA SEPARATED BZ MAPPINGS", help="Comma separated list of FEDORA_BZ=RHEL_BZ mappings.")
+        parser.add_argument("-s", "--skip-acks", dest="skip_acks", action="store_true", default=False,
+                            help="Skip checking for rhel-X.X.X ack flags.")
+        parser.add_argument("-S", "--skip-all", dest="skip_all_acks", action="store_true", default=False,
+                            help="Skip all checks.")
+        parser.add_argument("-d", "--debug", dest="debug", action="store_true", default=False,
+                            help="Enable debug logging to stdout.")
+        parser.add_argument("--zanata-branch", dest="zanata_branch", metavar="BRANCH NAME",
+                            help="Zanata branch name (needed if different from git branch name & not present in default mappings).")
+        parser.add_argument("--skip-zanata", dest="skip_zanata", action="store_true", default=False,
+                            help="Skip checking Zanata config for branch name.")
+        parser.add_argument("--skip-jenkins", dest="skip_jenkins", action="store_true", default=False,
+                            help="Skip checking Jenkins for test results.")
+        parser.add_argument("--dry-run", dest="dry_run", action="store_true", default=False,
+                            help="Do not change any files, only run checks.")
+
+        # gather all unprocessed command line arguments
+        parser.add_argument(nargs=argparse.REMAINDER, dest="unknown_arguments")
+
+        self.args = parser.parse_args()
+        if self.args.debug:
+            log.setLevel(logging.DEBUG)
+
+        if self.args.unknown_arguments:
+            sys.stderr.write("unknown arguments:%s" % self.args.unknown_arguments)
```

Comment:
I think it would be nice to print help here with some empty lines between error and help.